java/ql/src/utils/model-generator/RegenerateModels.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+# Tool to regenerate existing framework CSV models.
+
+from pathlib import Path
+import json
+import os
+import requests
+import shutil
+import subprocess
+import tempfile
+import sys
+
+
+lgtmSlugToModelFile = {
+    # "apache/commons-beanutils": "java/ql/lib/semmle/code/java/frameworks/apache/BeanUtilsGenerated.qll",
+    # "apache/commons-codec": "java/ql/lib/semmle/code/java/frameworks/apache/CodecGenerated.qll",
+    # "apache/commons-lang": "java/ql/lib/semmle/code/java/frameworks/apache/Lang3Generated.qll",
+    "apache/commons-io": "java/ql/lib/semmle/code/java/frameworks/apache/IOGenerated.qll",
+}
+
+
+def findGitRoot():
+    return subprocess.check_output(
+        ["git", "rev-parse", "--show-toplevel"]).decode("utf-8").strip()
+
+
+def regenerateModel(lgtmSlug, extractedDb):
+    tmpDir = tempfile.mkdtemp()
+    print("============================================================")
+    print("Generating models for " + lgtmSlug)
+    print("============================================================")
+    modelFile = lgtmSlugToModelFile[lgtmSlug]
```

Comment:
Maybe add some exception handling as we might have forgotten to update the lgtmSlugToModelFile, when adding entries to the strategy matrix in the workflow?